src/sentry/reprocessing.py

```diff
@@ -1,11 +1,32 @@
 from __future__ import absolute_import
 
 import uuid
+import logging
 
 
 REPROCESSING_OPTION = "sentry:processing-rev"
 
 
+logger = logging.getLogger("sentry.events")
+
+
+def event_supports_reprocessing(data):
+    """Only events of a certain format support reprocessing."""
+    from sentry.stacktraces.processing import find_stacktraces_in_data
+    from sentry.stacktraces.platform import NATIVE_PLATFORMS, JAVASCRIPT_PLATFORMS
+    platform = data.get("platform")
+    if platform in NATIVE_PLATFORMS:
```

Comment:
Minidump events should return false there, see:

https://github.com/getsentry/sentry/blob/d19a2aa395e9086d7c2dd9bddbff77f34bc83932/src/sentry/lang/native/error.py#L93-L98

We can use `event_supports_reprocessing` in that location then, as well.

Reply:
That's already checked in that function.

Reply:
I don't see it, where does `event_supports_reprocessing` call `is_minidump_event`?

Reply:
This should read: `if platform in NATIVE_PLATFORMS and not is_minidump_event(data):`

Reply:
I’m in phone atm but `report_processing_issue` internally checks.